webapp/src/main/webapp/iaf/gui/views/ShowScheduler.html

```diff
@@ -112,7 +112,7 @@ <h4>Jobgroup: {{::jobGroupName}}</h4>
 							<button ng-if="::job.type=='DATABASE'" title="Delete Job {{::job.name}}" class="btn btn-danger btn-xs" type="button" ng-click="remove(jobGroupName, job.name)"><i class="fa fa-times"></i> Delete</button>
 							<button ng-if="job.state=='PAUSED'" title="Resume Job {{::job.name}}" class="btn btn-warning btn-xs" type="button" ng-click="resume(jobGroupName, job.name)"><i class="fa fa-play"></i> Resume</button>
 							<button ng-if="job.state!='PAUSED'" title="Pause Job {{::job.name}}" class="btn btn-warning btn-xs" type="button" ng-click="pause(jobGroupName, job.name)"><i class="fa fa-pause"></i> Pause</button>
-							<button title="Trigger Job {{::job.name}}" class="btn btn-info btn-xs" type="button" ng-click="trigger(jobGroupName, job.name)"><i class="fa fa-arrow-circle-o-right"></i> Trigger</button>
+							<button ng-if="job.state!='PAUSED'" title="Trigger Job {{::job.name}}" class="btn btn-info btn-xs" type="button" ng-click="trigger(jobGroupName, job.name)"><i class="fa fa-arrow-circle-o-right"></i> Trigger</button>
```

Comment:
Why would it only be possible to trigger a job when it is not paused?

Reply:
Could you give us an estimate how much work it would be to implement a feature so it's possible to manually trigger a job even though it's paused? The pause should apply to the scheduled triggers not the manual ones.